beacon-chain/rpc/eth/beacon/pool.go

```diff
@@ -347,7 +350,7 @@ func (bs *Server) SubmitSignedBLSToExecutionChanges(ctx context.Context, req *et
 			toBroadcast = append(toBroadcast, alphaChange)
 		}
 	}
-	bs.Broadcaster.BroadcastBLSChanges(ctx, toBroadcast)
+	go bs.broadcastBLSChanges(ctx, toBroadcast)
```

Comment:
I worry about this design. So for every BLs message submitted, we will spawn a go routine, which terminates when the list is empty. I worry about submitting non-trivial BLS messages; the go routine count will spike up.

Why not just have one constant go routine and watch over a BLS message in a data structure?  `SubmitSignedBLSToExecutionChanges` inserts the BLS message into that data structure. The go routine can be a no-op if the data structure is empty?

Reply:
I feel the same way that the ideal situation would be to send this bls change over a channel, and the pool has a goroutine that processes incoming messages into batches

Reply:
It could be a single background routine in the pool that aggregates data received via a channel into batches and then broadcasts them based on a ticker to prevent spawning many routines

Reply:
Posting here cause we're having several discussions in parallell offline: this routine will be called once and never again in the vast majority of cases. The endpoint takes a list with all your changes. In the event some operator goes the length of writing his own tooling to try to Dos itself and get several lists with a single message in it, the routine will return earlier that the socket will be ready for the next call.

Reply:
> It could be a single background routine in the pool that aggregates data received via a channel into batches and then broadcasts them based on a ticker to prevent spawning many routines

Having a background routine for an event that should not happen ever seems like a higher impact. This endpoint should be called by the user once and never again.